PKHeX.WinForms/MainWindow/Main.cs

```diff
@@ -3838,7 +3839,18 @@ private void getSlotFiller(int offset, PictureBox pb)
             pb.Image = sprite;
             pb.BackColor = Color.Transparent;
             pb.Visible = true;
-        }
+			checkIfLegalPKMSlot(p, pb);
```

Comment:
Nit-pick: The new lines of code use tabs instead of spaces, making them look more indented than needed.